src/main/kotlin/io/github/paulgriffith/core/ThemeButton.kt

```diff
@@ -9,7 +9,12 @@ import javax.swing.Icon
 import javax.swing.JToggleButton
 import javax.swing.UIManager
 
-class ThemeButton : JToggleButton() {
+class ThemeButton(var isDark: Boolean = false) : JToggleButton() {
```

Comment:
By making this `var`, you're adding a mutable property to the class; that shouldn't be necessary. You can drop the `var` keyword and just pass a boolean in directly. The default value also isn't necessary since ThemeButton is only constructed in one place.

Reply:
Thanks for the insight Paul. I guess I was thinking that it might need a mutable property if we added a listener in the future, but that's overkill given the scope of the audience for this application.

Reply:
Yeah, I've been trying to stick to [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) on this project as much as possible, even though my own impulses really want me to do otherwise :)